skipper_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/zalando/skipper/filters/auth"
 	"github.com/zalando/skipper/filters/builtin"
 	fscheduler "github.com/zalando/skipper/filters/scheduler"
+	"github.com/zalando/skipper/innkeeper"
```

Comment:
Should we rather deprecate this?

Reply:
It's already since long time, but I need to use to to create the default data clients. I can refactor the code to drop it from test, but I don't really care for now.